packages/mongodb-memory-server-core/src/util/MongoInstance.ts

```diff
@@ -445,6 +450,7 @@ export class MongoInstance extends EventEmitter implements ManagerBase {
             con = await MongoClient.connect(uriTemplate(ip, port, 'admin'), {
               ...this.extraConnectionOptions,
               directConnection: true,
+              serverSelectionTimeoutMS: 500,
```

Comment:
```suggestion
              serverSelectionTimeoutMS: 1000, // 1 second
```

just to be safe, it will be 1 second